clientcmd/clientcmd.go

```diff
@@ -42,6 +46,39 @@ func NewClient() (*Client, error) {
 	}, nil
 }
 
+func (c *Client) PodLogs(namespace, podName string, opts *corev1.PodLogOptions) ([]string, error) {
+	logs, err := c.client.Pods(namespace).GetLogs(podName, opts).Stream(context.TODO())
+	if err != nil {
+		return nil, err
+	}
+	defer logs.Close()
+
+	logArr := make([]string, 0)
+	sc := bufio.NewScanner(logs)
+	for sc.Scan() {
+		logArr = append(logArr, sc.Text())
+	}
+	return logArr, errors.Wrap(sc.Err(), "reading logs stream")
+}
+
+func (c *Client) IsPodRunning(namespace, podName string) (bool, error) {
+	pod, err := c.client.Pods(namespace).Get(context.TODO(), podName, metav1.GetOptions{})
+	if err != nil {
+		return false, err
```

Comment:
And here?

Reply:
can't use wrapper there, since then we are checking error with `k8serrors.Is....`, which can cause issues